admin/test/test_installer.py

```diff
@@ -245,37 +279,49 @@ def _new_stack(self):
         environment variable. AWS credentials and CloudFormation parameter
         values must also be supplied as environment variables.
```

Comment:
Can you please consider updating this comment to reflect the fact that AWS credentials are now gathered from acceptance.yml? Thanks!